src/elements/checkbox/checkbox.spec.js

```diff
@@ -0,0 +1,122 @@
+describe('Semantic-UI: Elements - smCheckbox', function() {
+  'use strict';
+
+  var $scope, $compile, $rootScope;
+
+  beforeEach(module('semantic.ui.elements.checkbox'));
+
+  beforeEach(inject(function(_$rootScope_, _$compile_) {
+    $rootScope = _$rootScope_;
+    $scope = $rootScope.$new();
+    $compile = _$compile_;
+  }));
+
+  it('should transclude the text content', function() {
+    var smCheckbox = $compile('<sm-checkbox ng-model="test">Checkbox</sm-checkbox>')($scope);
```

Comment:
I think we should have a test where the transcluded content is an element transclusion directive, like `<sm-checkbox ng-model="test"><div ng-if="true">myLabel</div></sm-checkbox>` or something like that.

I think we'd also want to test sm-checkbox with ng-repeat on it. It's okay if they don't work right now, but try it and if they don't, please file bugs about making those work